config/_default/menus/menus.en.yaml

```
@@ -3887,7 +3887,7 @@ main:
     parent: ide_integrations
     identifier: ide_integrations_idea
     weight: 801
-  - name: Visual Studio Code
+  - name: VS Code
```

```suggestion
  - name: Visual Studio Code
```
From a style perspective, we follow Microsoft's own style for the product name, which from looking at their docs, is "full name first, then VS Code after." This will also help if someone searches on the term "Visual Studio". I'm updating your edits accordingly.